src/python/foglamp/admin_api/login.py

```diff
@@ -65,19 +58,21 @@
 
 @authentication_required
 async def get_user(request):
-    '''An example method that responds with the currently logged in user's details in JSON format'''
+    """An example method that responds with the currently logged in user's details in JSON format"""
     return json_response({'user': str(request.user)})
```

Comment:
The User class and that model.py file will be deleted when we decide how to map the user_info table in Python.